src/frequenz/sdk/timeseries/_quantities.py

```diff
@@ -301,6 +308,16 @@ def __call__(cls, *_args: Any, **_kwargs: Any) -> NoReturn:
         )
 
 
+class Temperature(
+    Quantity,
+    metaclass=_NoDefaultConstructible,
+    exponent_unit_map={
+        0: "°C",
+    },
+):
+    """A temperature quantity (in degrees Celsius)."""
```

Comment:
You should add a `from_celsius()` constructor and `as_celsius()` as in the other quantities, so users will always have to specify the used unit to create and to use a value.